daemon/cluster/executor/container/controller.go

```diff
@@ -1,16 +1,14 @@
 package container
 
 import (
-	"bufio"
-	"bytes"
-	"encoding/binary"
 	"fmt"
-	"io"
 	"os"
 	"strconv"
 	"strings"
 	"time"
 
+	"golang.org/x/time/rate"
```

Comment:
I removed this package and then added it back while I was hacking. I'll put it back where it came from.